source/reference/minio-mc/mc-anonymous-get-json.rst

```diff
@@ -1,7 +1,7 @@
 .. _minio-mc-policy-get-json:
 
 ======================
-``mc policy get-json``
+``mc anonymous get-json``
 ======================
 
```

Comment:
Apparently I can't put suggestions on "deleted" lines. Thanks git.

The top and bottom decorators need to expand enough to cover the full length of the new string. Need to do this across all of these pages.